app/wallet.py

```diff
@@ -90,7 +84,7 @@ def fromSk(self, sk, network):
   def fromBase58(self, address):
     addr = Wallet(address)
     if (not addr.isValid()):
-      logging.error(f'Invalid Ergo address ${address}')
+      return None
```

Comment:
returning None may cause NoneType exception if the caller expects return type as Wallet and checks the isValid method on None.